Orange/regression/pls.py

```diff
@@ -256,6 +255,10 @@ def incompatibility_reason(self, domain):
                     reason = "Only numeric target variables expected."
         return reason
 
+    def fitted_parameters(self, _) -> list[Learner.FittedParameter]:
+        return [self.FittedParameter("n_components", "Number of components",
```

Comment:
I think we should strive towards using the same name for a parameter in Parameter Fitter as it is labelled in the learner widget. In this case, PLS has "Components" and Param. fitter has "Number of components".